web/app.fluidity.money/app/routes/$network/dashboard/airdrop/common.tsx

```diff
@@ -986,7 +1039,7 @@ const StakeNowModal = ({
             className="power-text"
           >
             {toSignificantDecimals(
-              (parseFloat(fluidToken.amount) || 0) *
+              (2 * parseFloat(fluidToken.amount) || 0) *
```

Comment:
Should reflect the price of weth if it's in use, and don't multiply it, the pool could be made up however really